packages/nextcloud/lib/src/nextcloud.openapi.json

```diff
@@ -1894,47 +1894,29 @@
         }
       },
       "UserStatusStatus": {
-        "type": "object",
-        "required": [
-          "userId",
-          "messageIsPredefined",
-          "status",
-          "statusIsUserDefined"
-        ],
-        "properties": {
-          "userId": {
-            "type": "string"
-          },
-          "message": {
-            "type": "string"
-          },
-          "messageId": {
-            "type": "string"
-          },
-          "messageIsPredefined": {
-            "type": "boolean"
-          },
-          "icon": {
-            "type": "string"
+        "allOf": [
```

Comment:
@Leptopoda I should finalize my really outdated patch that adds inheritance

Reply:
That would be really neat :) With the automatic OpenAPI specs it will get a lot more important because it appears much more often there.